Fit/accessFit.py

```diff
@@ -1,36 +1,39 @@
-from deephaven import DynamicTableWriter, Types as dht
-from deephaven.DBTimeUtils import convertDateTime
+Heart_rateHeart_rateHeart_ratefrom deephaven import DynamicTableWriter, Types as dht
+from deephaven.DateTimeUtils import convertDateTime
 # Ensure fitparse is installed.
 import os
-os.system("pip install fitparse")
-from fitparse import FitFile
+try:
+    from fitparse import FitFile
+except ImportError:
+    os.system("pip install fitparse")
+    from fitparse import FitFile
 
 # Change to the name of the downloaded file (including any intermediate directory added to docker)
-fitfile = FitFile('/data/ThursMorn.fit')
+fitfile = FitFile('/data/examples/Fit/ThursMorn.fit')
 
 # See the size of your file
 records = list(fitfile.get_messages('record'))
-print("Number of data points:" + str(len(records)))
+print("Number of data points: {}".format(len(records)))
 
 # Setup deephaven tables to hold results
 # Heart rate
-columnNames = ["Timestamp", "Heart Rate"]
-columnTypes = [dht.datetime, dht.int_]
-hrTableWriter = DynamicTableWriter(columnNames, columnTypes)
-heartRateData = hrTableWriter.getTable()
+column_names = ["Timestamp", "Heart Rate"]
```

Comment:
Column names have spaces.  They should be camel case with a caps at the start and no spaces.